apps/dot_ext/forms.py

```diff
@@ -20,6 +20,11 @@ class CustomRegisterApplicationForm(forms.ModelForm):
                                   % (settings.APP_LOGO_SIZE_MAX, settings.APP_LOGO_WIDTH_MAX,
                                      settings.APP_LOGO_HEIGHT_MAX))
 
+    description_input = forms.CharField(label="Application Description",
```

Comment:
Is this indirection necessary in order to set things like `label`, etc. ?

Reply:
It is not necessary.  Was something I was doing in the admin form when the validation was still in the model, and I copied over.  I'll update this!